serverless/serverless-azure-functions
# WIP: Azure Function Deployment Slot

## What did you implement:

Closes #372



## How did you implement it:

* Default the slot to 'staging' in the DeploymentConfig
* Append the slot name to to the function name in the ARM template
* Allow the user to override the slot name --slot <name>



## How can we verify it:

* Generate a new functions project and deploy, the default deploy should be staging
* TBD: Now deploy to the production slot with --slot production



## Todos:

_**Note: Run `npm run test:ci` to run all validation checks on proposed changes**_

       **Validate via `npm run lint`**  
       _Note: Some reported issues can be automatically fixed by running `npm run lint:fix`_
       **Validate via `npm test`**

**_Is this ready for review?:_** NO  
**_Is it a breaking change?:_** NO
